sdk/cosmosdb/cosmos/package.json

```diff
@@ -133,7 +133,9 @@
     "sinon": "^9.0.2",
     "snap-shot-it": "^7.9.1",
     "source-map-support": "^0.5.9",
-    "ts-node": "^8.3.0",
+    "ts-node": "^9.0.0",
+    "tslint": "^5.0.0",
```

Comment:
aren't we using `eslint` instead?

Reply:
I don't really know how this happened, but I fixed it.